### ISSUE TITLE
sipp.dtd

### ISSUE THREAD

**Issue by akampitsch** [SIPp/sipp#303]
So i'm using the sipp.dtd that was packed with the source of v3.6-dev
Whenever i turn an interpreter on i get this error

Could not process schema or catalog: Open quote is expected for attribute "command" associated with an element type "exec". (happens in jEdit, ATOM with linter and Eclipse)

```
<!ELEMENT exec EMPTY >
<!-- one of command or int_cmd must be present -->
<!ATTLIST exec command CDATA >
<!ATTLIST exec int_cmd (stop_now|stop_gracefully|stop_call) "stop_call" >
<!ATTLIST exec play_pcap CDATA >
<!ATTLIST exec rtp_stream CDATA >
```

can any1 help me out with this?

**Comment by fleith** [comment]
Hi @akampitsch,
You can use the patch on https://github.com/SIPp/sipp/pull/306 and change the schema format to RELAX NG.

For that you need to change `<!DOCTYPE scenario SYSTEM "sipp.dtd">` to `<?xml-model href="sipp.rnc" type="application/relax-ng-compact-syntax"?>`


Follow the schema file:
[sipp.rnc.zip](https://github.com/SIPp/sipp/files/965578/sipp.rnc.zip)

**Comment by akampitsch** [comment]
While that works and gives me no errors in my editor(i had to edit a bit of the sipp.rnc)
now i can't use sipp with it, because the parser does not account for it and i end up with 2017-05-02	14:05:10.563298	1493726710.563298: No 'scenario' section in xml scenario file.

because it is not ignored the first element it looks at (should be <scenario name=""> is now <?xml-model ....?>

**Comment by fleith** [comment]
Yes, in your case you should fix dtd file or waiting for #31  or next version if #306 will accepted.

**Comment by TjlHope** [comment]
JIC it's helpful to anyone else, I modified the DTD to match the docs as closely as I could. (validation errors in vim/eclipse were bugging me too much!)
We probably use a reasonably old sipp version though, so double check before relying on it.
[sipp.dtd.txt](https://github.com/SIPp/sipp/files/2510880/sipp.dtd.txt)
